### ISSUE TITLE
Remove World Title and Nicknames from Published Build

### ISSUE THREAD

**Issue by alhardwarehyde** [hackclub/sinerider#197]
### Description

Good to keep for the dev build but shouldn't be in the final product.

### Screenshots

<img width="258" alt="Screenshot 2023-03-14 at 11 48 14 AM" src="https://user-images.githubusercontent.com/122998778/225058007-2468e9ae-aa19-41d8-8fa5-8fa953af726f.png">

**Comment by camdan-me** [comment]
I believe I saw in the Slack that y'all wanted to detect if the URL was `sinerider.com`, but can we use environment variables instead? That way we can test prod builds locally. @TheScientist101 thoughts?

**Comment by camdan-me** [comment]
Yeah I think I saw that, I'm saying an ENV variable might make more sense. It'd be a little harder to implement but it'd allow you to test the production build locally as well.

If you're using window.location.hostname, then you can't easily run the prod build on your local machine because you don't have access to sinerider.com.

**Comment by alhardwarehyde** [comment]
@polytroper thoughts on how to best implement this?

**Comment by TheScientist101** [comment]
Oh, sorry, I misunderstood your response. Sure, I can try to use an env variable, that would make more sense. @camdan-me

**Comment by camdan-me** [comment]
I'm not entirely sure how this would be implemented seeing as the website doesn't have any server-side code, but you can probably come up with something.

Or could even do something as simple as a `?development=true` tag to put it in dev mode, lots of options. Chris will know better.
